### ISSUE TITLE
Resolve string field references in filtered()

### ISSUE THREAD

**Issue by trinhanhngoc** [odoo-ide/pycharm-odoo#35]
```python
subscription.recurring_invoice_line_ids.filtered('product_id.storable_product_id')
```

**Comment by trinhanhngoc** [comment]
cc @jcfernandez-890825

**Comment by jcfernandez-890825** [comment]
Oh ... I get it. BaseModel.filtered doesn't allow to Ctrl+Click its fields, like with BaseModel.mapped.

Is that it?

**Comment by ghost** [comment]
> Oh ... I get it. BaseModel.filtered doesn't allow to Ctrl+Click its fields, like with BaseModel.mapped.
> 
> Is that it?

Yep!

**Comment by trinhanhngoc** [comment]
This was implemented in the version 2021.2.9.
